pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/queues.tsx

```diff
@@ -68,6 +68,13 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
+    <LineGraph title="Merge Queue" sources={storeSources}>
+    <Axis units={AxisUnits.Count} label="actions">
+      <Metric name="cr.store.queue.merge.process.success" title="Successful Actions / sec" nonNegativeRate />
+      <Metric name="cr.store.queue.merge.pending" title="Pending Actions" downsampleMax />
+    </Axis>
+    </LineGraph>,
```

Comment:
lint: indent `Axis`:

```js
    <LineGraph title="Merge Queue" sources={storeSources}>
      <Axis units={AxisUnits.Count} label="actions">
        <Metric name="cr.store.queue.merge.process.success" title="Successful Actions / sec" nonNegativeRate />
        <Metric name="cr.store.queue.merge.pending" title="Pending Actions" downsampleMax />
      </Axis>
    </LineGraph>,
```

Reply:
Oops, thanks. Done.

Reply:
Np. I should find the linter setting that catches stuff like this; seems like it's not on…